calculus/api.ts

```diff
@@ -48,4 +48,42 @@ export function evaluate_function(fx: string, variables : object) : APIResult {
             error: e.message
         }
     }
+}
+
+
+/**
+ * @param fx string expression of the function (example: x^2).
+ *          If null, empty or undefined, return InvalidParameterForFunction in error
+ * @param variable string corresponding to the variable according to which we want to compute the derivative
+ *          If null, empty or undefined, return InvalidVariableForFunction in error and the string expression of the
+ *          function in result
+ * @return json if there is no error, return the string corresponding to thr expression of the derivative in result
+ */
+export function first_simple_derivative(fx: string, variable: string) : object {
+    if (fx === undefined || fx == null){
```

Comment:
fx === undefined is included in fx == null

Reply:
i'll fix it ;)

Reply:
done ;)